Core/OxidEE_DB_Manager.php

```diff
@@ -0,0 +1,321 @@
+<?php
+/**
+ * Shop System Plugins - Terms of Use
```

Comment:
Please use this header for PHP files instead of the one you have currently
https://github.com/wirecard/opencart-ee/blob/master/catalog/controller/extension/payment/wirecard_pg_creditcard.php#L2-L8

Please make sure to add these two files as well
- https://github.com/wirecard/opencart-ee/blob/master/_TERMS_OF_USE
- https://github.com/wirecard/opencart-ee/blob/master/LICENSE